releases/release-1.31/release-notes/maps/pr-126163-map.yaml

```
@@ -0,0 +1,6 @@
+pr: 126163
+releasenote:
+  text: 'kube-apiserver: when the Alpha `UserNamespacesPodSecurityStandards` feature
+    gate is enabled, Pod Security Admission enforcement of the baseline policy now
+    allows `procMount=Unmasked` for user namespace pods that set `hostUsers=false`.'
```

```suggestion
    allows `procMount: Unmasked` for user namespace pods that set `hostUsers: false`.'
```
`procMount=Unmasked` is not a valid syntax in anything end users would recognize; we prefer actual syntax over pseudocode.